### DISCUSSION TITLE
may i know about the pricing please

### DISCUSSION THREAD

**Discussion by ankit01122** [spaceuptech/space-cloud#1560] [Q&A] [answered]
may i know about the pricing please of space cloud?

**Comment by YourTechBud** [comment, marked as answer]
Short answer: Its free. There is no pricing.

Longer answer: Since v0.21.4, SpaceCloud is fully open source and only offers community support.

**Comment by ankit01122** [comment]
What if in future we found company has made its software a paid one and we
are dependent on it?

**Reply by YourTechBud** [reply]
If I understand you correctly, you want to make a closed sourced software which you intend to sell and are concerned if using SpaceCloud in it will cause problems??

If that's the case, don't worry. You can use any Apache 2 licensed software (such as SC) in a commercial product without having to worry about royalty or any other challenges.

**Comment by ankit01122** [comment]
I mean there are softwares in market which are apache licensed but still
heavily paid.. What if space cloud moves to that category?

**Reply by YourTechBud** [reply]
You are referring to an open core model. The apache licensed version of a software is always free. However there can be commercial offering build on top of such tools which add additional features.

This repo will always be Apache licensed. Feel free to fork this repo to retain a copy just in case.

However, anyone (including you) is free to build a SC commercial offering if they desire. It will be upto you to stick with the open source version of the tool or opt for the commercial offering

**Reply by ankit01122** [reply]
anyway to install this on windows ? Because Istio is for linux only..

**Reply by YourTechBud** [reply]
For local development, you can use wsl2 and docker desktop: https://docs.spaceuptech.com/install/kubernetes/docker-desktop/

In production however, you will need to set k8s up on linux
